docs-app/package.json

```diff
@@ -126,7 +126,7 @@
   },
   "dependencies": {
     "@crowdstrike/ember-oss-docs": "^1.1.3",
-    "@crowdstrike/ember-toucan-core": "workspace:../ember-toucan-core",
+    "@crowdstrike/ember-toucan-core": "workspace:../packages/ember-toucan-core",
```

Comment:
I believe `workspace:*` would be all we need here.

Reply:
You're right! Just updated.  Thank you!